openedx/core/djangoapps/content/course_overviews/tests.py

```diff
@@ -164,6 +166,11 @@ def get_seconds_since_epoch(date_time):
             self.assertEqual(course_value, cache_miss_value)
             self.assertEqual(cache_miss_value, cache_hit_value)
 
+        # test tabs against course
+        course_overview_tabs = course_overview_cache_hit.course_overview_tab.all()
+        course_resp_tabs = [tab.tab_id for tab in course_overview_tabs]
+        self.assertEqual(self.COURSE_OVERVIEW_TABS, course_resp_tabs)
```

Comment:
Change the above 2 lines so the comparison is a `set` comparison instead of a `list` comparison (since order shouldn't matter).